.github/workflows/orchestration-dev.yml

```diff
@@ -8,14 +8,17 @@ on:
       - opened
       - reopened
       - synchronize
+      - ready_for_review
 
 jobs:
   analysis:
+    if: github.event.pull_request.draft == false
```

Comment:
we don't want draft PRs deployed to dev for the high risk that they will break because they aren't ready